monkey/monkey_island/cc/environment/utils.py

```diff
@@ -35,10 +37,14 @@ def _create_secure_directory(path: str, create_parent_dirs: bool):
         raise ex
 
 
-def set_secure_permissions(dir_path: str):
+def _create_secure_directory_windows(path: str):
     try:
-        if is_windows_os():
-            windows_permissions.set_perms_to_owner_only(folder_path=dir_path)
+        security_attributes = win32security.SECURITY_ATTRIBUTES()
+        security_attributes.SECURITY_DESCRIPTOR = windows_permissions.get_sd_for_owner_only_perms()
```

Comment:
"Dnt abbrev"

Generally, avoid abbreviations unless they're very common. `get_security_descriptor_for_owner_only_perms()` would be a better name, since I don't have to wonder what `sd` means.